exporters/BxDRobotExporter/RobotExporterTests/obj/RobotExporterTests.csproj.nuget.dgspec.json

```diff
@@ -1,34 +1,34 @@
 {
   "format": 1,
   "restore": {
-    "C:\\Users\\Victo\\Documents\\Sythesis\\synthesis\\exporters\\BxDRobotExporter\\RobotExporterTests\\RobotExporterTests.csproj": {}
+    "C:\\Users\\t_espin\\Development\\nuget\\synthesis\\exporters\\BxDRobotExporter\\RobotExporterTests\\RobotExporterTests.csproj": {}
```

Comment:
@matthew17754 Should this file be included if it has the full path to users' repo folder?

Reply:
It doesn't seem to affect building at all, but I feel like a lot if not all files in the "obj" folder aren't necessary to include in source control. I'll experiment and see what I can figure out.